modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportDDG2P.pm

```diff
@@ -210,13 +247,14 @@ sub parse_input_file {
         push @phenotypes, {
           'id' => $gene->stable_id,
           'description' => $phen,
-          'external_id' => $id,
+          'MIM' => $id,
           'seq_region_id' => $gene->slice->get_seq_region_id,
           'seq_region_start' => $gene->seq_region_start,
           'seq_region_end' => $gene->seq_region_end,
           'seq_region_strand' => $gene->seq_region_strand,
           'mutation_consequence' => $mode,
           'inheritance_type' => $allelic,
+          'pubmed_id'  => @pubmeds,
           'accessions' => \@accns,
```

Comment:
Still part of the code to add the confidence and the mutation consequence:

```
'g2p_confidence' => $confidence_category,
'mutation_consequence' => $mutation_consequence,
```

Note: I used the attrib_type_id as keys, in my example my ids are 'g2p_confidence' and 'mutation_consequence'. You have to use the keys that are going to be inserted in homo_sapiens_variation_108_38.